content/docs/capabilities/metrics.md

```diff
@@ -1,79 +1,266 @@
+Changes can be auto-applied (the default behavior), or applied manually. If you apply changesets manually, you can review pending changes before applying them to your configuration:
 
-1. In the Pomerium Enterprise `pomerium-enterprise-config.yaml`, define the `metrics_addr` key to a network interface and/or port. For example:
+![Applying a changeset in Pomerium Zero](./img/metrics/zero-changesets.png)
 
-   ```yaml title="config.yaml"
-   metrics_addr: 127.0.0.1:9092
-   ```
+After you've applied a change or a changeset, you can review them in the Deployments dashboard:
 
-## External Prometheus
+![Reviewing changesets in Pomerium Zero](./img/metrics/zero-deployments-changesets.gif)
 
-1. Add the listener to your Prometheus configuration, usually via `prometheus.yml`:
+</TabItem>
+<TabItem label="Enterprise" value="enterprise">
 
-   ```yaml
-   - job_name: 'Pomerium'
-      scrape_interval: 30s
-      scrape_timeout: 5s
-      static_configs:
-         - targets: ['127.0.0.1:9901']
-   - job_name: 'Pomerium Enterprise'
-      scrape_interval: 30s
-      scrape_timeout: 5s
-      static_configs:
-         - targets: ['127.0.0.1:9902']
+From the **Deployments** page, administrators can review changes made to their Pomerium configuration.
 
-   ```
+The default view shows all changes made through Pomerium Enterprise. Use the **COMPARE** button next to an entry to filter to only changes that affected that resource. Select two versions of that resource, then **DIFF** to see what changed:
 
-1. [Reload](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#configuration) the Prometheus configuration:
+![A screenshot showing the diff of a change to a route, adding a policy](./img/reports/reports-deployments-diff.png)
 
-   ```bash
-   curl -i -XPOST path.to.prometheus:port/-/reload
-   ```
+</TabItem>
+</Tabs>
 
-1. In the Pomerium Enterprise `config.yaml` file, define the `prometheus_url` key to point to your Prometheus instance(s):
+## Runtime (Enterprise)
 
-   ```yaml
-   prometheus_url: http://192.168.122.50:9090
-   ```
+In the **Runtime** dashboard, you can monitor how many system resources Pomerium is consuming. Filter by date range, service, and instance.
 
-1. Restart the Pomerium and Pomerium Enterprise services. You should now see route traffic and External Data Source monitoring data in the Enterprise Console:
+![The Runtime Info page in Pomerium Enterprise](./img/metrics/reports-runtime-fullpage.png)
 
-   ![Traffic Data in Pomerium Enterprise](./img/metrics/console-route-traffic.png) ![External Data Source in Pomerium Enterprise](./img/metrics/console-ext-datasource-monitoring.png)
+## Sessions (Enterprise)
```

Comment:
This section is unrelated to metrics and need be moved to a separate section.

Reply:
Same question as above.